docs/conf.py

```diff
@@ -63,9 +64,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '0.1'
+version = pkg_resources.get_distribution("nidmm").version
```

Comment:
Please either document why this is the way it is or open an issue that explains it.

Reply:
Added comments.

Also changed `version` to be X.Y with `release` being the entire string